<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editComputers</class>
 <widget class="QDialog" name="editComputers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit a computer</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HorizontalLayoutGender_2">
    <item>
     <widget class="QLabel" name="labelGender_2">
      <property name="text">
       <string>Was it built? </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxBuilt">
      <item>
       <property name="text">
        <string>Yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacerGender_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutDeath_2">
    <item>
     <widget class="QLabel" name="labelDeath_2">
      <property name="text">
       <string>Year Created:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditComputerYearCreated"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutBirth_2">
    <item>
     <widget class="QLabel" name="labelBirth_2">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditComputerType"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutName_2">
    <item>
     <widget class="QLabel" name="labelName_2">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditComputerName"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textComputerEditInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listComputersCreators">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listComputersPeopleRest">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>235</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Creators:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Other Scientists in DB:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
